src/documentation/wiki/prototyping-deck.md

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
This should eventually be part of the product pages: https://www.bitcraze.io/products/prototyping-deck/. Make sure to add a link to those page as well for the technical details. Eventually for #93, we will try to fix the product pages to be more practical

Reply:
Agreed, and fixed.